packages/react/src/components/Accordion/AccordionItem.tsx

```diff
@@ -196,6 +196,7 @@ function AccordionItem({
       </Toggle>
       <div
         ref={content}
+        hidden={!isOpen}
```

Comment:
Would it be more appropriate to use `aria-hidden` here? Since `hidden` prevents the contents from being rendered at all, searches on collapsed content would not return any results. Although searching through collapsed, `aria-hidden` content will not expand it, it will at least show a result in the browser.

Reply:
Actually, disregard this. I just tested and `aria-hidden` still allows tab focus even when `true`.